singolo/style.css

```diff
@@ -0,0 +1,718 @@
+  }
+
+  .slider-content {
+    width: 100%;
+    height: 100%;
+  }
+
+  .phone-vertical {
+    width: 180.71px;
+    height: 363.67px;
+    margin-right: 19px;
+  }
+
+  .phone-horizontal {
+    width: 363.67px;
+    height: 180.71px;
+  }
+
+  .icon-previous-page {
+    width: 16px;
+    height: 28.31px;
+    margin-right: 3px;
+  }
+
+  .icon-next-page {
+    width: 16px;
+    height: 28.31px;
+  }
+
+  .services {
+    padding-top: 39.62px;
+    padding-bottom: 64px;
+    border-bottom: none;
+  }
+
+  .services .wrapper {
+    padding: 0 41px 0 42px;
+  }
+
+  .services-list {
+    display: grid;
+    grid-template-columns: 1fr 1fr;
+    grid-row-gap: 39px;
+    grid-column-gap: 53px;
+    padding-top: 28px;
+  }
+
+  .portfolio {
+    max-height: 1094.4px;
+    padding-bottom: 65.4px;
+    border-bottom: 7.6px solid #323746;
+  }
+
+  .portfolio-list {
+    max-width: 1019px;
+    grid-template-columns: 1fr 1fr 1fr;
+    grid-gap: 19.65px 19.65px;
+  }
+
+  .portfolio .wrapper {
+    padding: 0 41.13px 0 39px;
+  }
+
+  .portfolio .wrapper > h2 {
+    margin-bottom: 14px;
+  }
+
+  .portfolio .wrapper > p {
+    margin-bottom: 12px;
+  }
+
+  .portfolio .wrapper > ul {
+    margin-bottom: 28px;
+  }
+
+  .portfolio .wrapper > ul > li {
+    margin-right: 10px;
+    padding: 4px 5px 6px;
+  }
+
+  .footer {
+    padding: 22px 0;
+  }
+
+  .footer .wrapper {
+    padding: 0 42px 0 39px;
+  }
+  .footer .footer-content > .footer-icons > li:first-child {
+    margin-left: 0;
+  }
+}
+
+@media screen and (max-width: 767px) {
+  .header-content {
+    justify-content: center;
+  }
+
+  .hamburger-menu {
+    display: block;
+  }
+
+  .menu-btn {
+    display: flex;
+    align-items: center;
+    position: fixed;
+    top: 26px;
+    left: 20px;
+    width: 26px;
+    height: 26px;
+    cursor: pointer;
+    z-index: 2;
+    border-radius: 5px;
+    background-color: #2d303a;
+  }
+
+  .menu-btn > span,
+  .menu-btn > span::before,
+  .menu-btn > span::after {
+    display: block;
+    position: absolute;
+    width: 100%;
+    height: 2.4px;
+    background-color: #ffffff;
+  }
+  .menu-btn > span::before {
+    content: "";
+    top: 8px;
+  }
+  .menu-btn > span::after {
+    content: "";
+    top: -8px;
+  }
+
+  .toggled-menu-box {
+    display: block;
+    position: fixed;
+    visibility: hidden;
+    top: 0;
+    left: -100%;
+    width: 100%;
+    height: 100%;
+    margin: 0;
+    padding: 0;
+    background-color: rgba(45, 48, 58, 0.5);
+    transition: left ease 0.5s;
+  }
+
+  .menu-box {
+    display: block;
+    position: fixed;
+    top: 0;
+    width: 278px;
+    height: 100%;
+    margin: 0;
+    padding: 0;
+    list-style: none;
+    text-align: center;
+    background-color: #2d303a;
+    box-shadow: 5px 5px 10px #2d303a;
+  }
+
+  .header nav ul {
+    padding: 0;
+  }
+
+  .menu-item {
+    display: block;
+    padding: 27px 0 0 71px;
+    color: #ffffff;
+    font-size: 24px;
+    line-height: 29px;
+    font-weight: 700;
```

Comment:
Which `font-weight` value do you have for the element for screens bigger then 767px?